asc2018/vdr/electrical/WaterlooMidnightSun_ASC2018VDR_BatteryTechReport.tex

```diff
@@ -0,0 +1,119 @@
+\theauthor \\
+\theauthorcontact \\
+\fi
+\thedate \\
+\vfill
+\theteamwebsite \\
+\theteamphone
+\end{titlepage}
+
+% Main Matter
+\tableofcontents
+%\listoffigures % <-------------- uncomment for list of figures
+%\listoftables % <-------------- uncomment for list of tables
+
+\section{Battery Specifications}
+
+\begin{table}[!htbp]
+\centering
+\begin{tabular}{|l|l|l|}
+\hline
+\multirow{4}{*}{5.2.D.1} & Manufacturer Name                     & LG Chemical Limited                                                                                                                   \\ \cline{2-3}
+                         & Manufacturer Contact                  & \begin{tabular}[c]{@{}l@{}}Jon Caserta, Liion Wholesale Batteries (Supplier)\\ support@liionwholesale.com\\ 888-972-2883\end{tabular} \\ \cline{2-3}
+                           & Cell Specification URL                & \url{http://vruzend.com/SpecificationINR18650MJ1.pdf}                                                                                       \\ \cline{2-3}
+                         & Cell MSDS URL                         & \url{http://bigship.com/telechargement?doc=148635}                                                                                          \\ \hline
+5.2.D.2                  & Stock Number                          & INR18650 MJ1                                                                                                                          \\ \hline
+5.2.D.3                  & Cell \& Module Voltage                & \begin{tabular}[c]{@{}l@{}}3.635 V (nominal)\\ 4.200 V (max)\end{tabular}                                                               \\ \hline
+5.2.D.4                  & Bus Voltage                           & \begin{tabular}[c]{@{}l@{}}130.86 V (nominal)\\ 151.2 V (max)\end{tabular}                                                              \\ \hline
+\multirow{3}{*}{5.2.D.5} & Cells per module                      & 36                                                                                                                                    \\ \cline{2-3}
+                         & Modules per string                    & 36                                                                                                                                    \\ \cline{2-3}
+                         & Strings in parallel                   & 1                                                                                                                                     \\ \cline{2-3}
+                         & Total cell count                      & 1296                                                                                                                                  \\ \hline
+  \multirow{3}{*}{5.2.D.6} & Capacity (kWh)                        & $\frac{3.5\text{Ah} \times 130.86\text{V} \times 36}{1000} = 16.48836$                                                                            \\ \cline{2-3}
```

Comment:
NIT: The units should have a single space between them and the value. Might be worthwhile adding units to the 36 and 1000 for clarity.